docs/fleet/provision.md

```diff
@@ -1,171 +1,157 @@
-  {{< /alert >}}
-
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl stop viam-agent
-  ```
-
-- To restart the Viam Agent:
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "manufacturer": "Skywalker",
+  "model": "C-3PO",
+  "fragment_id": "2567c87d-7aef-41bc-b82c-d363f9874663",
+  "hotspot_prefix": "skywalker-setup",
+  "disable_dns_redirect": true,
+  "hotspot_password": "skywalker123",
+  "roaming_mode": false,
+  "offline_timeout": "3m30s",
+  "user_timeout": "2m30s",
+  "fallback_timeout": "15m"
+}
+```
 
-  {{< alert title="Alert" color="note" >}}
-  When you restart the Viam Agent, the agent will restart `viam-server` as well.
-  {{< /alert >}}
+This file configures some basic metadata, specifies a [fragment](/fleet/fragments/) to use to configure the machine, and provides the WiFi hotspot network name and password to use on startup.
+It also configures timeouts to control how long the Agent waits for a valid local WiFi network to come online before creating its hotspot network, and how long to keep the hotspot active before terminating it.
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl restart viam-agent
-  ```
+{{% /tab %}}
+{{< /tabs >}}
 
-- To completely uninstall the Viam Agent and `viam-server`, run the following command:
+{{% expand "Run additional provisioning steps with a pre-install script" %}}
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/uninstall.sh)"
-  ```
+When you install `viam-agent`, either manually using the commands above or automatically as part of your fleet's build and deploy process, you can choose to run a pre-install script to perform provisioning steps before deployment to the target machine.
+For example, you could use the pre-install script to configure and deploy to an SD card or other image file, which you can then use as part of your fleet deployment process.
+You can also use this method to generate a local tarball containing the configured deployment, which you could then deploy later, or through a different medium (such as automation, or as the basis for further customer-specific customization).
 
-  This command uninstalls Viam Agent, `viam-server`, the machine configuration file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).
+For more information, see [Pre-installed provisioning](https://github.com/viamrobotics/agent-provisioning#pre-installed-provisioning).
+{{% /expand%}}
 
-  For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
+### Attributes
 
-### View Viam Agent logs
+<!-- prettier-ignore -->
+| Name       | Type   | Required? | Description |
+| ---------- | ------ | --------- | ----------- |
+| `manufacturer` | string | Optional | Purely informative. May be displayed on captive portal and/or mobile app. Default: `"viam"`. |
```

Comment:
```suggestion
| `manufacturer` | string | Optional | Purely informative. May be displayed on captive portal and/or mobile app. Default: `"viam"`. |
```
do we use and/or? don't know how else i would put it but see this part is WIP

Reply:
```suggestion
| `manufacturer` | string | Optional | Purely informative. May be displayed on captive portal or mobile app. Default: `"viam"`. |
```